src/split-panel/index.tsx

```diff
@@ -338,11 +339,17 @@ export default function SplitPanel({
     }
   }, [lastInteraction]);
 
-  const wrappedChildren = (
+  const visualRefreshContext = useContext(VisualRefreshAppLayoutContext);
+
+  const wrappedChildren = isRefresh ? (
+    <VisualRefreshAppLayoutContext.Provider value={{ ...visualRefreshContext, offsetBottom: 0 }}>
+      {children}
+    </VisualRefreshAppLayoutContext.Provider>
+  ) : (
     <AppLayoutContext.Provider
       value={{
         stickyOffsetTop: topOffset,
-        stickyOffsetBottom: bottomOffset,
+        stickyOffsetBottom: 0,
```

Comment:
Just making sure - even though you've added the ternary for the `VisualRefreshAppLayoutContext`, do we also want this value to be always `0` ?

Reply:
Thank you pointing out, it is better to solve it on table. Updated in the new revision.